Makefile

```diff
@@ -8,6 +8,7 @@
 qa:
 	isort src/structurizr/ tests/ examples/ setup.py
 	black src/structurizr/ tests/ examples/ setup.py
+	flake8 src/structurizr/ tests/ examples/ setup.py
```

Comment:
I will remove this line before merging. `make qa` is intended for applying style fixes and not for checking for problems. You can either use `tox -e flake8` for that or create a separate command `make qc`.

Reply:
OK, sure.